Backend/server.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express'
+import { MongoClient } from 'mongodb'
+import { url } from './database/db.js'
+
+require('dotenv').config({ path: '../.env' })
```

Comment:
Does using dotenv effectively replace passing in environment variables via the command line when starting up the app?

Reply:
[Yes](https://www.npmjs.com/package/dotenv)

Reply:
I'm open to other implementations. This is all about us figuring out how we want to develop this.

Reply:
No, this looks good. I had read through the npm package but I wasn't sure from it whether that meant you can't pass in environment variables. Having a single file for them is a gooooood solution.

Reply:
👍 Apologies if the 'Yes' came across passive aggressive! Not intended.